templates/pytorch/src/__ALGO__.py

```diff
@@ -0,0 +1,57 @@
+import Algorithmia
+import torch
+from . import helpers
+from PIL import Image
+import numpy as np
+
+TARGET_IMAGE_SIZE = 224
+CLASSES = ['cat', 'dog']
```

Comment:
nit: we're using single and double quotes in this file. I think we should only commit to one: single quotes?